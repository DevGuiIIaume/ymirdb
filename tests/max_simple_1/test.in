set a -10 - 5 0 5 10
max a
set b 100
max b
bye
