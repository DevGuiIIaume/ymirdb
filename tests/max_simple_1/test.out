> ok

> 10

> ok

> 100

> bye
